src/dlls/mscorpe/mscorpe/wrapper.cpp

```diff
@@ -60,6 +60,14 @@ HRESULT
 LoadMscorpe(HMODULE * phModule)
 {
```

Comment:
We should statically link this link into ilasm (or into coreclr.dll), so that we do not need to deal with hassles on how/where to distribute it. It is very small amount of code.